### PR TITLE
chore(readme): Improved readme! :wrench:

### LINKED ISSUE #632: Forbedring av README

### PR DESCRIPTION
**Pull request by norgor**
## Proposed changes
READMEen er nå mer konsis og lettere å lese.

Issue number: closes #632

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] The fixtures have been updated if needed (for migrations)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,10 @@
 </p>
 
 
-<h1 align="center">LEPTON</h1>
+<h1 align="center">Lepton</h1>
 
 <h4 align="center">
-    TIHLDEs open source backend for <a href="https://tihlde.org">tihlde.org</a> - the organizations main web site.
+    Open source backend for <a href="https://tihlde.org">TIHLDE's website.</a>
```
Comment (martcl) [inline README.md:12]: Her brukes det <a> taggs mens under brukes det readme-brackets til å lenke. Vi kan holde oss til en stil.

**Review by martcl — APPROVED** [review]
Streak🔥

File: README.md (hunk 1 above)
Comment (norgor) [reply README.md:12]: Jeg har prøvd, men det går ikke inne i HTML-tags :sob:

Comment (martcl) [reply README.md:12]: 😢

**Pull request merged**